doc/man/8/rados.rst

```diff
@@ -97,6 +97,8 @@ Pool specific commands
 
 :command:`put` *name* *infile* [--offset offset]
   Write object name with start offset (default:0) to the cluster with contents from infile.
+  
+  **Warning:** Please be aware that this directly creates a single object at the RADOS layer directly. This can interfere with data distribution across the OSDs and even overload the cluster when trying to replicate it. Consider using the frontend protocols like S3, RBD, CephFS instead, which all stripe the data properly.
```

Comment:
I would say someting like "...if the object is more than a few 10s of MB".  Small objects are fine.